test/openjd/cli/test_run_command.py

```diff
@@ -295,7 +327,10 @@ def test_do_run_success(
         )
 
         # WHEN
-        do_run(args)
+        try:
+            do_run(args)
+        except:  # noqa: E722
+            pass
```

Comment:
Does this have a chance of obfuscating any real test failures?

Reply:
Good point. I'll narrow it to trapping only SystemExit -- which is what gets raised in the test cases that early-exit intentionally.